variables.tf

```diff
@@ -162,4 +162,9 @@ variable "cbr_zones" {
   default = []
 }
 
+variable "skip_cloud_shell_calls" {
+  type        = bool
+  description = "Enable/Disable Global Cloud Shell settings in the account."
```

Comment:
This description is wrong - its not enabling or disabling anything - its skipping the setting all together